### PR TITLE
Pull request 

### PR DESCRIPTION
**Pull request by Sartish**
Link to my deployed project:
https://saras-todos.netlify.app/

My reflections on how this week's project turned out:
Difficult to understand all the new concepts completely. But better understanding of the "redux flow" But I need more practise 

Things I'd like to have clarified or explained in more detail:
Maybe what the redux toolkit actually help us with, what would the alternative look like, if this didn't help us

### CODE REVIEW

File: code/src/reducers/todos.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { createSlice } from '@reduxjs/toolkit'
```
Comment (nazi-pira) [inline code/src/reducers/todos.js]: Hi Sara!
I really enjoyed having a look at your clean, organized code and I have learned a lot as well! The only point can be getting rid off comments provided by the teachers! Good job!

**Comment by nazi-pira** [comment]
Would also like to have a look at the live page but the link does not work for me!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+The app also includes: Clear all function, input field and category selector for new tasks which are saved in local storage 
 
-## View it live
+🤯Here I struggled a bit with how to set ut up in a correct way so data can be changed through dispatching the action-payload to the store. This therefore involved googling solutions, looking at other peoples solution and trying to understand the overall flow of redux. This helped me  a lot when writing the redux actions and then using that data in your components.
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+⏰ In this project was the first time using both Redux toolkit and material ui so there is a lot left there to learn. But in this project if I had more time I would actually work more on understanding the flow. Not understanding the complete flow makes the code a bit more messier than needed. 
+
+### Tech ⚛️
+👉Redux
```
Comment (KarolCislowski) [inline README.md:21]: You asked about the toolkit and vanilla redux. You can read what toolkit makes for you here: [https://redux.js.org/redux-toolkit/overview](https://redux.js.org/redux-toolkit/overview ) 
It is a very complicated process to configure your own store in Redux so if you do not need to have some exotic setting it is more convenient to use the toolkit 😸

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,36 @@
...
+  persistedState = JSON.parse(persistedStateJSON);
+};
+
+// So now we setup the store with the reducer we defined and the persistedState
+// as the initial state. The store subscribes to any actions performed on the state
+// so that it always stores the most updated data
+const store = createStore(reducer, persistedState, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
+
+store.subscribe(() => {
+  localStorage.setItem("reduxState", JSON.stringify(store.getState()));
```
Comment (KarolCislowski) [inline code/src/App.js:27]: Persistence 😻

File: code/src/components/AddNewTodo.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
+import { TextField } from '@material-ui/core'
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { Fab, Select, MenuItem , InputLabel, FormControl } from '@material-ui/core';
```
Comment (KarolCislowski) [inline code/src/components/AddNewTodo.js]: If you are importing more than 2 components from one file, then organize your list of components in a column to make it easier to read and debug 🤓

**Review by KarolCislowski — APPROVED** [review]
Nice app with a great, colorful interface 😻

**Pull request closed without merging**